src/libpriv/rpmostree-kernel.c

```diff
@@ -211,52 +208,140 @@ rpmostree_find_kernel (int rootfs_dfd,
   return g_variant_ref_sink (g_variant_new ("(sssms)", kver, bootdir, kernel_path, initramfs_path));
 }
 
-/* Given a kernel path and a temporary initramfs, compute their checksum and put
- * them in their final locations.
+/* Given a @rootfs_dfd and path to kernel/initramfs that live in
+ * usr/lib/modules/$kver, possibly update @bootdir to use them.
+ * @bootdir should be one of either /usr/lib/ostree-boot or /boot.
+ * If @copy_if_not_found is set, we do the copy unconditionally,
```

Comment:
The parameter is still called `is_auto`, though I prefer `copy_if_not_found` as well. :) Or maybe keep it reversed and name it something like `only_if_found`?